### ISSUE TITLE
Quest 878 text order incorrect...

### ISSUE THREAD

**Issue by The-Great-Sephiroth** [classicdb/database#525]
When you speak to the captured quillboar Mangletooth, he starts by thanking you for helping him instead of his correct text. It is the text which is supposed to come after his entire questline is complete. You may still start the questline, but it is odd that he is asking for bloodshards before you are introduced to them.

**Comment by cala** [comment]
Thanks for this report. Could you rephrase this issue or detail it a bit more, please?
I looked at the texts of quest 878 and I saw no reference to bloodshards in them.
Did you mean that the NPC gossip menu is wrong?
